test/node-unit/cli/config.spec.js

```diff
@@ -17,6 +16,17 @@ describe('cli/config', function() {
   });
 
   describe('loadConfig()', function() {
+    let parsers;
```

Comment:
this "unit" test was reading files, and it shouldn't need to read files to assert what's trying to assert.